src/topology/algebra/continuous_functions.lean

```diff
@@ -372,3 +378,50 @@ instance continuous_map_module' {α : Type*} [topological_space α]
 end continuous_map
 
 end module_over_continuous_functions
+
+/-!
+We now provide formulas for `f ⊓ g` and `f ⊔ g`, where `f g : C(α, β)`,
+in terms of `continuous_map.abs`.
+-/
+
+section
+variables {R : Type*} [linear_ordered_field R]
+
+-- This lemma (and the next) could go all the way back in `algebra.ordered_field`,
```

Comment:
This makes me sad, but not sad enough to try for an alternate proof 😉 .
```suggestion
-- TODO: This lemma (and the next) could go all the way back in `algebra.ordered_field`,
```

Reply:
Done.